plugins/clnrest/utilities/shared.py

```diff
@@ -81,6 +81,8 @@ def convert_millisatoshis(item):
         ret = [convert_millisatoshis(i) for i in item]
     elif isinstance(item, pyln.client.Millisatoshi):
         ret = int(item)
+    elif isinstance(item, str) and item.endswith("msat"):
+        ret = int(item[:-4])
```

Comment:
No, try making an invoice with description "fromsat" and watch clnrest crash :(

Reply:
Resolved by @rustyrussell.